### ISSUE TITLE
Accessibility Highlight

### ISSUE THREAD

**Issue by arobillard** [catherine9/animal-rescue-website#5]
The accessibility highlight (black outline) is not showing up in your nav bar because of a lack of contrast. I recommend changing it to another color. The below code should work as a base.

```
.masthead a:focus {
  outline: 3px solid #fff;
}
```
